packages/compass-import-export/src/components/export-select-fields.tsx

```diff
@@ -0,0 +1,261 @@
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+
+import {
+  Body,
+  Button,
+  Icon,
+  Table,
+  TableHeader,
+  Row,
+  Cell,
+  TextInput,
+  css,
+  spacing,
+  Checkbox,
+  Disclaimer,
+} from '@mongodb-js/compass-components';
+
+const headerContainerStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  marginBottom: spacing[3],
+  marginTop: spacing[3],
+  gap: spacing[2],
+});
+
+const tableContainerStyles = css({
+  maxHeight: spacing[7] * 3,
+  overflow: 'auto',
+});
+
+const checkboxContainerStyle = css({
+  display: 'flex',
+});
+
+const cellContainerStyles = css({
+  padding: `${spacing[1]}px ${spacing[2]}px`,
+});
+
+const inputCellContainerStyles = css(cellContainerStyles, {
+  verticalAlign: 'middle',
+});
+
+const smallCellContainerStyle = css(cellContainerStyles, {
+  width: spacing[5],
+  margin: '0 auto',
+});
+
+const textInputStyles = css({
+  padding: `${spacing[1]}px 0`,
+  minWidth: spacing[7] * 3,
+});
+
+const enterToAddStyles = css({
+  display: 'inline-flex',
+  alignItems: 'center',
+  marginLeft: spacing[2],
+});
+
+type ExportSelectFieldsProps = {
+  fields: Record<string, number>;
```

Comment:
I feel like this should be a boolean, it's only ever 1 or 0. It touches a bit more code than this though, so if folks like it I can update in the other places in the package. https://github.com/mongodb-js/compass/blob/b9d3ece378f90236697a6a3fbe84a820072af15f/packages/compass-import-export/src/modules/load-fields.ts#L61

Reply:
Ended up going fully for boolean, I think it's more consistent with how we do variables in the rest of the app.